requirements.txt

```diff
@@ -87,9 +87,9 @@ pika==1.3.1
 pymongo==4.3.3
 
 # DBT
-dbt-bigquery==1.4.0
-dbt-core==1.4.0
-dbt-postgres==1.4.0
+dbt-bigquery==1.4.3
```

Comment:
You also need to update the dbt packge version in setup.py
* https://github.com/mage-ai/mage-ai/blob/master/setup.py#L121
* https://github.com/mage-ai/mage-ai/blob/master/setup.py#L64

Reply:
updated on d27c638831d274c104381dcc25fde217b37e0a86